internal/conf/configuration.go

```diff
@@ -241,8 +241,8 @@ type GlobalConfiguration struct {
 	Metrics                 MetricsConfig
 	SMTP                    SMTPConfiguration
 	RateLimitHeader         string  `split_words:"true"`
-	RateLimitEmailSent      float64 `split_words:"true" default:"30"`
-	RateLimitSmsSent        float64 `split_words:"true" default:"30"`
+	RateLimitEmailSent      Rate    `split_words:"true" default:"30"`
+	RateLimitSmsSent        Rate    `split_words:"true" default:"30"`
```

Comment:
do we want to keep the default setting here to 30?

Reply:
Yup, no breaking changes